lib/app/util.js

```diff
@@ -13,4 +13,8 @@ export function findPageForPath (pages, path) {
       return page
     }
   }
+  return {
+    path: '',
+    frontmatter: {}
+  }
```

Comment:
Is it to fix some bugs, or?

Reply:
Without this, there will be errors rendering `NotFound.vue`, as there is not `$page` match it.

Reply:
In fact it to fix a bug. I'll extract it to another PR

Reply:
#388